utils/releasetools/03_test_release.sh

```diff
@@ -1,26 +1,28 @@
 #!/bin/sh
+set -e
 if [ $# != "1" ]
 then
-    echo "Usage: ${0} <git-ref>"
+    echo "Usage: utils/releasetools/03_test_release.sh <version_tag>"
     exit 1
 fi
 
 TAG=$1
 TARNAME="redis-${TAG}.tar.gz"
 DOWNLOADURL="http://download.redis.io/releases/${TARNAME}"
 
-ssh antirez@metal "export TERM=xterm;
-                   cd /tmp;
-                   rm -rf test_release_tmp_dir;
-                   cd test_release_tmp_dir;
-                   rm -f $TARNAME;
-                   rm -rf redis-${TAG};
-                   wget $DOWNLOADURL;
-                   tar xvzf $TARNAME;
-                   cd redis-${TAG};
-                   make;
-                   ./runtest;
-                   ./runtest-sentinel;
-                   if [ -x runtest-cluster ]; then
-                       ./runtest-cluster;
-                   fi"
+echo "Doing sanity test on the actual tarball"
+
+cd /tmp
+rm -rf test_release_tmp_dir
+cd test_release_tmp_dir
+rm -f $TARNAME
+rm -rf redis-${TAG}
+wget $DOWNLOADURL
+tar xvzf $TARNAME
+cd redis-${TAG}
+make
+./runtest
+./runtest-sentinel
+./runtest-cluster
+./runtest-moduleapi
```

Comment:
Consider building with TLS and running sanity twice, with and without `--tls`?

Reply:
@yossigo the real tests that "approve" the release are the ones in the daily CI (include LIBC malloc, 32-bit, valgrind, etc).

also note that in the past, this step was running in some reserved (consistent) bare metal machine, and now i changed it to just run on the local machine.

i intend for this test step to be just an extra sanity (one that actually tests the released tarball). in that respect maybe i should remove the cluster tests which IIRC take a long time.